src/pages/index.js

```diff
@@ -119,45 +181,129 @@ function closeModalEscape(evt) {
 
 function handleEditFormSubmit(evt) {
   evt.preventDefault();
-  profileName.textContent = editModalNameInput.value;
-  profileDescription.textContent = editModalDescriptionInput.value;
-  closeModal(editModal);
+  const submitButton = evt.submitter
+  submitButton.textContent = "Saving...."
+
+  api
+  .editUserInfo({ name: editModalNameInput.value,
+    about: editModalDescriptionInput.value
+  })
+  .then((userData) => {
+
+    profileName.textContent = userData.name;
+    profileDescription.textContent = userData.about;
+    closeModal(editModal);
+  })
+  .catch(console.error)
+  .finally(() => {
+   submitButton.textContent = "Save";''
+
+  });
+
 }
 
-function handleAddCardSubmit(evt) {
+function handleSubmit(evt) {
```

Comment:
`handleSubmit` in `index.js` should be named `handleAddCardSubmit` because it's for submitting a card form

I was talking only about the function that was inside `helpers.js`